README.md

```diff
@@ -1,9 +1,48 @@
 # CAN-Logger-3
 
-An open source data logging system for heavy vehicle networks.
+An open source secure data logging system for heavy vehicle networks. This project is comprensive in that it covers the following design features:
+  1. Hardware electronics design using and ARM Cortex M4F 32-bit processor. This includes schematics, bill of materials, and Altium design files. The design is based on the Teensy 3.6 from PJRC.
+  2. The embedded software stack using the Arduino and Teensyduino development environments.
+  3. A Python 3 application running PyQT5 to interface with the embedded device.
+  4. A microservices-based web backend with Amazon Web Services using Python and the serverless frameworks.
+
+## Goals
+The end use of the device is to securely log all the vehicle network traffic on a heavy vehicle. Some of the stated goals:
+  1. Reliably capture all network data when connected to a vehicle on multiple vehicle networks to include 3 CAN channels and J1708.
+  2. Securely store all the collected data in non-volitile memory. This means all the data collected will be encrypted before written to the SD card.
+  3. The encryption key will be ephemeral, which means each data set has its own key for each session. 
+  4. The ephemeral key is envelope encrypted using unique key generated for each device. If one of these keys is compromized, then only one device is compromised.
```

Comment:
to
"  4. The ephemeral key is envelope encrypted using a unique key generated for each device. If one of these keys is compromised, then only one device is compromised."